azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -75,34 +75,38 @@
 public class AzkabanExecutorServer {
   private static final String CUSTOM_JMX_ATTRIBUTE_PROCESSOR_PROPERTY = "jmx.attribute.processor.class";
   private static final Logger logger = Logger.getLogger(AzkabanExecutorServer.class);
-  private static final int MAX_FORM_CONTENT_SIZE = 10 * 1024 * 1024;
 
   public static final String JOBTYPE_PLUGIN_DIR = "azkaban.jobtype.plugin.dir";
   public static final String METRIC_INTERVAL = "executor.metric.milisecinterval.";
-  public static final int DEFAULT_HEADER_BUFFER_SIZE = 4096;
 
   private static final String DEFAULT_TIMEZONE_ID = "default.timezone.id";
-  private static final int DEFAULT_THREAD_NUMBER = 50;
 
   private static AzkabanExecutorServer app;
 
   private final ExecutorLoader executionLoader;
   private final FlowRunnerManager runnerManager;
   private final Props props;
   private final Server server;
+  private final Context root;
 
   private final ArrayList<ObjectName> registeredMBeans = new ArrayList<ObjectName>();
   private MBeanServer mbeanServer;
 
   @Inject
   public AzkabanExecutorServer(Props props,
       ExecutorLoader executionLoader,
-      FlowRunnerManager runnerManager) throws Exception {
+      FlowRunnerManager runnerManager,
+      @Named("ExecServer") Server server,
+      @Named("root") Context root) throws Exception {
```

Comment:
To call this on startup, and to avoid circular dependency:

```java
    root.setAttribute(Constants.AZKABAN_SERVLET_CONTEXT_KEY, this);
```

I think the proper way to refactor this would be to not make `AzkabanExecutorServer` depend on `Server` either, but I'm not sure if that can be easily done.